src/plugins/1/accessors/sdk/index.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: full.html.hbs
+title: sdk
+---
+
+# sdk
+
+{{{since "1.6.0"}}}
+
+Accessor to the embedded SDK. 
+
+The embedded SDK is a custom version of our [Javascript SDK]({{ site_base_path }}sdk-reference/js/6) that uses a custom protocol plugged directly into Kuzzle core.  
+
+Plugin developers can use every controllers except the `realtime` controller and also the [query]({{ site_base_path }}sdk-reference/js/6/kuzzle/query/) method of the SDK.
+
+### Request context
+
+By default, when using the embedded SDK, requests made to Kuzzle API don't have the same context as the original request received by the plugin.  
+
+Typically, the `request.context.user` property is not correctly set and thus [Kuzzle metadata]({{ site_base_path }}guide/1/essentials/document-metadata) will not be properly set when creating or updating documents.
```

Comment:
By definition, a documentation cannot state that there are  "incorrect" behaviors in a product: it should always state what the desired behavior should be. And if such problems do exist in the product, then tickets must be written for these issues to be resolved.
At most, a documentation can talk about limitations, and eventually offer workarounds to those.

In our case, the behavior described here is the desired one: a plugin is not the same entity than the user at the origin of a request. There can be various reasosn for a plugin to execute an API action, and most often than not, it's for its own reasons.

So you should change that whole section tone to reflect that, by default, a plugin executing an API action do so in its own name, but that the exposed SDK offer a way for the plugin to impersonate the querying user, with the `as()` method.